packages/server/database/migrations/20221123161224-updateTier.ts

```diff
@@ -0,0 +1,181 @@
+import {R} from 'rethinkdb-ts'
+import getPg, {closePg} from '../../postgres/getPg'
+
+export const up = async function (r: R) {
+  const BATCH_SIZE = 1000
```

Comment:
We need to batch when running a migration that affects many fields because rethink could timeout. There are several examples in our codebase where the batch size is 3000. As I'm updating multiple different tables in one `Promise.all`, I thought I'd go with a batch size of 1000. 

This is a fairly random number though and I was wondering if there's a better way to test what would be a good batch size and when we need to batch a migration?